xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/AdminWikisRightsSheet.xml

```diff
@@ -40,6 +40,7 @@
 ### Sheet used to generically display the XWikiPreferences object fields in the administration sheets.
 {{html}}
   &lt;form method="post" action="$xwiki.getURL($currentDoc, 'saveandcontinue')" class="xform"&gt;
+    &lt;input type="submit" name="submit" class="hidden" /&gt;
```

Comment:
Same comment as on https://github.com/xwiki/xwiki-platform/pull/912/files#r229593000 . Moreover, Bootstrap has the ``sr-only`` CSS class that we could use, if we want the screen readers to see the submit button.

Reply:
ok, thanks for the tip